src/pages/Loans/LoansOverview/components/BorrowLimit/BorrowLimit.tsx

```diff
@@ -4,63 +4,62 @@ import { useTranslation } from 'react-i18next';
 
 import { formatNumber, formatUSD } from '@/common/utils/utils';
 import { DlGroup, Dt } from '@/component-library';
+import { useAccountBorrowLimit } from '@/pages/Loans/LoansOverview/hooks/use-get-account-borrow-limit';
 import { LoanAction } from '@/types/loans';
-import { useGetAccountLoansOverview } from '@/utils/hooks/api/loans/use-get-account-loans-overview';
 
+import { useLoansHealthFactor } from '../../hooks/use-get-account-health-factor';
 import { getStatus, getStatusLabel } from '../../utils/get-status';
+import { isBorrowAsset } from '../../utils/is-loan-asset';
 import { LoanScore } from '../LoanScore';
 import { StyledAlert, StyledDd, StyledDl, StyledWarningIcon } from './BorrowLimit.style';
 
 const LIQUIDATION_ALERT_SCORE = 1.5;
 
 type BorrowLimitProps = {
   variant: LoanAction;
-  asset: MonetaryAmount<CurrencyExt>;
+  assetAmount: MonetaryAmount<CurrencyExt>;
   shouldDisplayLiquidationAlert?: boolean;
 };
 
-const BorrowLimit = ({ variant, asset, shouldDisplayLiquidationAlert }: BorrowLimitProps): JSX.Element => {
+const BorrowLimit = ({ variant, assetAmount, shouldDisplayLiquidationAlert }: BorrowLimitProps): JSX.Element => {
   const { t } = useTranslation();
-  const {
-    getNewCollateralRatio,
-    getNewBorrowLimitUSDValue,
-    data: { borrowLimitUSDValue }
-  } = useGetAccountLoansOverview();
 
-  const newBorrowLimit = getNewBorrowLimitUSDValue(variant, asset.currency, asset)?.toNumber() || 0;
-  const newCollateralRatio = getNewCollateralRatio(variant, asset.currency, asset) || 0;
-  const status = getStatus(newCollateralRatio);
-  const statusLabel = getStatusLabel(status);
+  const { data: borrowLimitUSD, getBorrowLimitUSD } = useAccountBorrowLimit();
+  const { getHealthFactor } = useLoansHealthFactor();
```

Comment:
It would be better to also return current health factor from `useLoansHealthFactor` in the same way the `borrowLimit` is returned from `useAccountBorrowLimit`, so we keep the hooks consistent.